smart-contracts/contracts/mixins/MixinKeys.sol

```diff
@@ -31,11 +31,21 @@ contract MixinKeys is
 
   event KeyManagerChanged(uint indexed _tokenId, address indexed _newManager);
 
-  // Keys
-  // Each owner can have at most exactly one key
-  // return 0 values when missing a key
+  // DEPREC: dont use
   mapping (address => Key) internal keyByOwner;
 
+  // store all keys: tokenId => token
+  mapping(uint256 => Key) private _keys;
+  
+  // store ownership: owner => list of tokenIds
+  mapping(address => mapping(uint256 => uint256)) private _ownedKeyIds;
```

Comment:
Do we really need `uint256` for the index for a user's token ids?
I wonder if we could use something significantly smaller and/or if it matters?

```suggestion
  mapping(address => mapping(uint256 => uint256)) private _ownedKeyIds;
```

Reply:
yes I think it is better to keep all in uint, it does not make much difference as we are not doing calculations on it